Dockerfile

```
@@ -1,14 +1,14 @@
-FROM golang:alpine AS build
+FROM --platform=$BUILDPLATFORM golang:1.23 AS build
 ARG COMMIT_SHA
 ARG VERSION
 ARG TARGETPLATFORM
 WORKDIR /build
 COPY . .
-RUN apk add build-base pkgconfig re2-dev bash
-RUN export GOARCH=${TARGETPLATFORM#*/} && bash build.sh $VERSION $COMMIT_SHA
+RUN apt-get update && apt-get install -y libre2-dev && if [ "$TARGETPLATFORM" == "linux/arm64"] ; then apt-get install -y crossbuild-essential-aarch64;else apt-get install -y build-essential; fi
```

Hadolint (2.12.0)

[error] Missing space before ]. Fix any mentioned problems and try again.

(SC1072)